src/controllers/payment.controller.ts

```diff
@@ -33,12 +36,18 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     // Validate the status of the payment
     if (status === PAYMENT_PAID) {
-      logger.debugRequest(req, `Overseas Entity id: ${ appData[OverseasEntityKey] }, Payment status: ${status}, Redirecting to: ${CONFIRMATION_URL}`);
+
+      let confirmationPageUrl = CONFIRMATION_URL;
+      if (isActiveFeature(FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
+
+        confirmationPageUrl = getUrlWithParamsToPath(CONFIRMATION_WITH_PARAMS_URL, req);
```

Comment:
looking at the url that is provided to the payment service to be used to call back to our service, it is using /transaction/:transactionId/overseas-entity/:overseaEntityId/ which is different to what our other pages have been using (using submissionId instead of overseaEntityId, so we'll have to think what to do with this case.
we could just get the params from teh url separately or change the callback url that is passed into teh payment service

Reply:
I'll create a new subtask for this and pick it up